frontend/src/components/smart/CharacterDisplay.vue

```diff
@@ -293,7 +306,7 @@ export default Vue.extend({
           a.isSelected = true;
         }
         else {
-          a.isSelected = false;
+          a.isSelected = true;
```

Comment:
If both branches of this `if` statement do the same thing, shouldn't they be unified?